posthog/clickhouse/migrations/0031_event_properties_zstd.py

```diff
@@ -0,0 +1,9 @@
+from infi.clickhouse_orm import migrations
+
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+operations = [
+    migrations.RunSQL(
+        f"ALTER TABLE sharded_events ON CLUSTER '{CLICKHOUSE_CLUSTER}' MODIFY COLUMN properties VARCHAR CODEC(ZSTD(3))"
```

Comment:
Probably irrelevant these days but maybe use `EVENTS_DATA_TABLE()`? Else we can also clean that up to a static value to avoid confusion

Reply:
We can now clean that up as we made the replication migration mandatory last release.